src/Styles/card.css

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 8px;
+}
+
+.card__photo {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 40px;
+  padding-top: 16px;
```

Comment:
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 9th item, you should use margins for indents between elements, not paddings, also you should either use margin-top or margin-bottom